api/scpca_portal/metadata_file.py

```diff
@@ -76,6 +76,25 @@ def load_library_metadata(metadata_file_path: Path):
         return transform_keys(json.load(raw_file), LIBRARY_METADATA_KEYS)
 
 
+def transform_metadata_dict(dict) -> Dict:
+    """
+    Returns the transformed dict after converting its value to a joined string from the list
+    """
+    for k, v in dict.items():
+        if isinstance(v, list):
+            dict[k] = utils.string_from_list(v)
+    return dict
+
+
+def transform_metadata_dicts(list_of_dicts: List[Dict]) -> List[Dict]:
+    """
+    Loops through and transforms the given list of dictionaries using transform_metadata_dict
+    """
+    for dict in list_of_dicts:
+        transform_metadata_dict(dict)
+    return list_of_dicts
```

Comment:
You should drop this in favor of the for loop in the `csv_writer` context manager

Reply:
Dropped at fa899d8092b18f0d64284f5fef20819e15fe0e75.